server/graphql/general/src/sync_api_error.rs

```diff
@@ -162,7 +165,7 @@ mod test {
                 source: ParsedError {
                     code: SyncErrorCodeV5::SiteIncorrectHardwareId,
                     message: "n/a".to_string(),
-                    data: Some("n/a".to_string()),
+                    data: Some(json!("n/a")),
```

Comment:
This was changed, it looks like `String` type for `data` was failing to deserialize from object